web/src/pages/apd/proposed-budget/CombinedActivityCosts.js

```diff
@@ -71,24 +71,39 @@ const SummaryBudgetByActivityTotals = ({ data, ffy }) => {
         </tr>
       </thead>
       <tbody>
-        <tr className="budget-table--row__header">
-          <th scope="row" colSpan="2">
-            HIT
-          </th>
-        </tr>
-        <DataRowGroup data={data.hit} year={ffy} groupTitle="HIT" />
-        <tr className="budget-table--row__header">
-          <th scope="row" colSpan="2">
-            HIE
-          </th>
-        </tr>
-        <DataRowGroup data={data.hie} year={ffy} groupTitle="HIE" />
-        <tr className="budget-table--row__header">
-          <th scope="row" colSpan="2">
-            MMIS
-          </th>
-        </tr>
-        <DataRowGroup data={data.mmis} year={ffy} groupTitle="MMIS" />
+        {data?.hit && (
```

Comment:
This is fine for now. I may change the conditionals to explicitly check the apdType rather than if the budget has .hit or not.